src/hammer-vlsi/synthesis/vivado/defaults.yml

```diff
@@ -0,0 +1,20 @@
+# Default settings for synthesis in Vivado, for project/technology configuration and overriding.
+synthesis.vivado:
+  # Location of the binary.
+  binary: "vivado"
+  # Location of the vivado setup script
+  setup_script: "/tools/projects/erichang/hammer-examples/vivado_setup.sh"
```

Comment:
Seems like we should check some stuff in?

Reply:
I think hammer has a way to reference files relative to the tool dir (please poke me about this)